go.mod

```diff
@@ -33,6 +33,7 @@ replace (
 
 replace (
 	github.com/edgelesssys/constellation/v2/operators/constellation-node-operator/v2/api => ./operators/constellation-node-operator/api
+	github.com/google/go-sev-guest v0.7.1 => github.com/msanft/go-sev-guest v1.1.0
```

Comment:
instead of forking, I'd prefer to `require` the latest (or whatever desired) commit from main